lua/lspconfig/lspinfo.lua

```diff
@@ -38,14 +43,14 @@ return function ()
     return cmd
   end
 
+  local indent = "  "
   local function make_client_info(client)
     return {
       "",
-      "Client: "..tostring(client.id),
-      "\tname: "..client.name,
-      "\troot: "..client.workspaceFolders[1].name,
-      "\tfiletypes: "..table.concat(client.config.filetypes or {}, ', '),
-      "\tcmd: "..remove_newlines(client.config.cmd),
+      indent.."Client: "..client.name.." (id "..tostring(client.id)..")",
```

Comment:
I mean that you add these empty spaces after client/root/filetype/cmd to align them.